### ISSUE TITLE
Deprecation after upgrading to ember-cli 2.6.2

### ISSUE THREAD

**Issue by ghost** [mmun/ember-cli-envy#1]
```
DEPRECATION: Overriding init without calling this._super is deprecated. Please call `this._super.init && this._super.init.apply(this, arguments);` addon: `ember-cli-envy`
    at Function.Addon.lookup (/Users/ben/src/recipe-builder/node_modules/ember-cli/lib/models/addon.js:896:27)
```

**Comment by mmun** [comment]
Oops, I was meaning to fix this. But feel free to remove ember-cli-envy from our Zesty projects. I, somewhat surprisingly, haven't needed to use it once yet! I originally made it to handle our deployment ENV vars but the continuous deployment + reverting deploys with `git revert` has been working great.

**Comment by ghost** [comment]
😮  I thought it was doing loads of stuff.

**Comment by halfdan** [comment]
Fixed by #2 after which I realised I won't need it.. but here we go.
